lib/presentation/screen/journal_detail_screen.dart

```diff
@@ -42,16 +48,23 @@ class JournalDetailScreen extends StatelessWidget {
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
             SizedBox(height: screenHeight * 0.06),
-            MoodWidget(date: date, moodColor: moodColor, mood: mood),
+            MoodWidget(date: date, moodColor: Colors.green, mood: mood),
             SizedBox(height: screenHeight * 0.04),
-            DocumentWidget(title: 'Title', description: titleDescriptoin),
+            DocumentWidget(title: 'Title', description: titleDescription),
             SizedBox(height: screenHeight * 0.02),
             DocumentWidget(title: 'Note', description: noteDescription),
           ],
         ),
       ),
       floatingActionButton: PrimaryFloatingActionButton(
-          onTap: () {}, child: const Icon(Icons.edit_outlined)),
+          onTap: () => context.goNamed('JournalEditScreen', params: {
+                'date': '05 September 2022',
+                // 'detailMoodColor': ,
+                'mood': 'Productive',
+                'titleDescription': 'Feeling Productive',
+                'noteDescription': 'Feeling Productive Feeling Productive',
```

Comment:
for edit params will be optional. because, if no param means new journals, if params means edit. or should we create seperate screens?

Reply:
I didn't get it